lib/utils/misc.js

```diff
@@ -1,3 +1,5 @@
+var SSC_COOKIE_NAME = '_td_ssc_id'
```

Comment:
There are many constants in td-js-sdk which, IMHO, should be collected to utils/constants.js file.

This is just a suggestion going forward. it's up to you whether to do it in this PR or not. I understand this PR is urgent :)